src/Controller/AbstractActionController.php

```diff
@@ -85,26 +87,4 @@ public function onDispatch(MvcEvent $e)
 
         return $actionResponse;
     }
-
-    /**
-     * @deprecated please use the {@see \Zend\Mvc\Controller\Plugin\CreateHttpNotFoundModel} plugin instead: this
-     *             method will be removed in release 2.5 or later.
-     *
-     * {@inheritDoc}
-     */
-    protected function createHttpNotFoundModel(HttpResponse $response)
```

Comment:
Actually they were removed in #99 
I will revert the commit on this for release-2.7 tho.